homework/Anastasia_Kichaeva/Hw5_Task5/src/main/java/acya/ki/creditCard/bean/CreditCard.java

```diff
@@ -0,0 +1,144 @@
+package acya.ki.creditCard.bean;
+
+import java.io.Serializable;
+import java.util.Scanner;
+
+public class CreditCard implements Serializable, Runnable {
+    private int cardNumber;
+    private int cardPinCod;
+    private double cardBalance;
+    private double creditLimit;
+    private double creditDebt;
+
+    public CreditCard() {
+    }
+
+    public CreditCard(int cardNumber, int cardPinCod) {
+        this.cardNumber = cardNumber;
+        this.cardPinCod = cardPinCod;
+        this.cardBalance = 0;
+        this.creditDebt = 0;
+        this.creditLimit = 0;
+    }
+
+    public int getCardNumber() {
+        return cardNumber;
+    }
+
+    public void setCardNumber(int cardNumber) {
+        this.cardNumber = cardNumber;
+    }
+
+    public int getCardPinCod() {
+        return cardPinCod;
+    }
+
+    public void setCardPinCod(int cardPinCod) {
+        this.cardPinCod = cardPinCod;
+    }
+
+    public double getCardBalance() {
+        return cardBalance;
+    }
+
+    public void setCardBalance(double cardBalance) {
+        this.cardBalance = cardBalance;
+    }
+
+    public double getCreditLimit() {
+        return creditLimit;
+    }
+
+    public void setCreditLimit(double creditLimit) {
+        this.creditLimit = creditLimit;
+    }
+
+    public double getCreditDebt() {
+        return creditDebt;
+    }
+
+    public void setCreditDebt(double creditDebt) {
+        this.creditDebt = creditDebt;
+    }
+
+    @Override
+    public String toString() {
+        return "CreditCard{" +
+                "cardNumber=" + cardNumber +
+                ", cardPinCod=" + cardPinCod +
+                ", cardBalance=" + cardBalance +
+                ", creditLimit=" + creditLimit +
+                ", creditDebt=" + creditDebt +
+                '}';
+    }
+
+    public void deposit() {
+        System.out.println("Input Pin cod: ");
+        Scanner scanner = new Scanner(System.in);
+        int pinCodForDeposit = scanner.nextInt();
+        if (pinCodForDeposit == cardPinCod) {
+            System.out.println("Input amount to be credited: ");
+            double depositAmount = scanner.nextDouble();
+            if (creditDebt == 0) {
+                cardBalance = cardBalance + depositAmount;
+            } else if (creditDebt < 0 && creditDebt + depositAmount <= 0) {
+                creditDebt = creditDebt + depositAmount;
```

Comment:
Credit debt should be >=0, because it's a sum that you owed to a bank. Thus when you balance's lower than 0, your debt is higher then 0